__tests__/app.test.js

```diff
@@ -45,4 +45,43 @@ describe("/api/topics", () => {
             expect(response.body.msg).toBe("Not Found!");
           });
       });
+});
+
+describe("/api", () => {
+    test("GET: 200 sends an array of endpoints to the client", () => {
+        return request(app)
+          .get("/api")
+          .expect(200)
+          .then((response) => {
+            expect(response.body).toEqual({
+                "GET /api": {
+                  "description": "serves up a json representation of all the available endpoints of the api"
+                },
+                "GET /api/topics": {
+                  "description": "serves an array of all topics",
+                  "queries": [],
+                  "exampleResponse": {
+                    "topics": [{ "slug": "football", "description": "Footie!" }]
+                  }
+                },
+                "GET /api/articles": {
+                  "description": "serves an array of all articles",
+                  "queries": ["author", "topic", "sort_by", "order"],
+                  "exampleResponse": {
+                    "articles": [
+                      {
+                        "title": "Seafood substitutions are increasing",
+                        "topic": "cooking",
+                        "author": "weegembump",
+                        "body": "Text from the article..",
+                        "created_at": "2018-05-30T15:59:13.341Z",
+                        "votes": 0,
+                        "comment_count": 6
+                      }
```

Comment:
A reasonable test for now, though as you're going to add to this endpoint as you progress, could do with a more flexible test. Every time you add to this endpoint, you would need to change this test
You could instead import the endpoints.json file and reference that in your tests